packages/lodestar/src/chain/blocks/importBlock.ts

```diff
@@ -128,9 +137,31 @@ export async function importBlock(chain: ImportBlockModules, fullyVerifiedBlock:
         }
         pendingEvents.push(ChainEvent.attestation, attestation);
       } catch (e) {
-        chain.logger.error("Error processing attestation from block", {slot: block.message.slot}, e as Error);
+        // a block has a lot of attestations and it may has same error, we don't want to log all of them
+        if (e instanceof ForkChoiceError && e.type.code === ForkChoiceErrorCode.INVALID_ATTESTATION) {
+          let errWithCount = invalidAttestationErrorsByCode.get(e.type.err.code) as
+            | {error: Error; count: number}
+            | undefined;
```

Comment:
Why do you need to type the return of get() here?

Reply:
yeah we really don't need to 👍